packages/@markuplint/rule-textlint/src/index.spec.ts

```diff
@@ -8,19 +8,19 @@ import path from 'path';
 
 test('is test 1', async () => {
 	const r = await markuplint.verify(
-		`<!DOCTYPE html>
-		<html lang="en">
-		<head>
-			<meta charset="UTF-8">
-			<meta name="viewport" content="width=device-width, initial-scale=1.0">
-			<meta http-equiv="X-UA-Compatible" content="ie=edge">
-			<title>Document</title>
-		</head>
-		<body>
-			<h1>Title</h1>
-			<p>This not use jquery.</p>
-		</body>
-		</html>`,
+		/* HTML */ `<!DOCTYPE html>
```

Comment:
I add `/* HTML */` to format because when I copy-paste the next test case, it results in different indentations in my editor, it is strange if the two test cases results different lint messages.